akka-http-core/src/main/scala/akka/http/javadsl/ConnectionContext.scala

```diff
@@ -47,7 +47,7 @@ object ConnectionContext {
    *  are enabled as needed.
    */
   @ApiMayChange
-  def httpsClient(createEngine: akka.japi.function.Function2[String, Int, SSLEngine]): HttpsConnectionContext =
+  def httpsClient(createEngine: akka.japi.function.Function2[String, Integer, SSLEngine]): HttpsConnectionContext =
```

Comment:
Did that become a BoxedInteger in Java? Or couldn't it be used at all?

Reply:
It became `Object` on the Java side, so it couldn't be used without casting.